.env.development

```
@@ -2,5 +2,5 @@ HTTPS=true
 SSL_KEY_FILE=certs/localhost-key.pem
 SSL_CRT_FILE=certs/localhost-cert.pem
 REACT_APP_PROXY_TARGET=https://backend.historycode.online/api
-REACT_APP_BACKEND_URL=/api
+REACT_APP_BACKEND_URL=https://localhost:5001/api
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REACT_APP_BACKEND_URL key should go before the REACT_APP_PROXY_TARGET key

(UnorderedKey)